src/content/1.7/development/webservice/tutorials/advanced-use/manage-multishop.md

```diff
@@ -0,0 +1,142 @@
+
+| Result | API call | PHP Webservice lib options |
+|--------|----------|----------------------------|
+| List `shops` | `/api/shops/` | {{< code php >}}$opt = [
+    'resource' => 'shops'
+];{{< /code >}} |
+
+## Define shop specific override
+
+To deal with shop specific values you can use the regular APIs and specify the `shop` or `shop_group` you are targeting. They can be used for read and/or write operations.
+
+| Key               | Value         |
+|-------------------|---------------|
+| **id_shop**       | Shop id       |
+| **id_shop_group** | Shop group id |
+
+## Create shop
+
+You can refer to the tutorial explaining how to [create a resource]({{< ref "1.7/development/webservice/tutorials/prestashop-webservice-lib/create-resource" >}}) or [update a resource]({{< ref "1.7/development/webservice/tutorials/prestashop-webservice-lib/update-resource" >}}) to add/update a `shop`. It also will need a `shop_url` otherwise using its `id_shop` will result in a redirection by the API.
+
+```php
+<?php
+
+require_once('./vendor/autoload.php');
+
+$webServiceUrl = 'http://example.com/';
+$webServiceKey = 'ZR92FNY5UFRERNI3O9Z5QDHWKTP3YIIT';
+$webService = new PrestaShopWebservice($webServiceUrl, $webServiceKey, false);
+
+// Check shop presence
+$shopName = 'Additional shop';
+$searchedShop = $webService->get(['resource' => 'shops', 'filter[name]' => $shopName]);
+$shopId = null;
+if ($searchedShop->shops->shop->count() > 0) {
+    $shopId = (int) $searchedShop->shops->shop[0]->attributes()['id'];
+    echo 'Shop already exists' . PHP_EOL;
+    die(1);
+}
+
+// Create shop
+$blankXml = $webService->get(['url' => $webServiceUrl . 'api/shops?schema=blank']);
+$shopXml = $blankXml->shop[0];
+$shopXml->name = 'Additional shop';
+$shopXml->id_shop_group = 1; // Default shop group
+$shopXml->id_category = 2; // Default category Root
+$shopXml->theme_name = 'classic';
+$shopXml->active = 1;
+
+$createdShop = $webService->add(['resource' => 'shops', 'postXml' => $blankXml->asXML()]);
+$shopId = (int) $createdShop->shop->id;
+echo 'Successfully created shop ' . $shopId . PHP_EOL;
+
+// Create shop url
+$blankXml = $webService->get(['url' => $webServiceUrl . 'api/shop_urls?schema=blank']);
+$shopUrlXml = $blankXml->shop_url[0];
+$shopUrlXml->id_shop = $shopId;
+$shopUrlXml->active = 1;
+$shopUrlXml->main = 1;
+$shopUrlXml->domain = 'example.com';
+$shopUrlXml->domain_ssl = 'example.com';
+$shopUrlXml->physical_uri = '/';
+$shopUrlXml->virtual_uri = '/additional';
+
+$createdShopUrl = $webService->add(['resource' => 'shop_urls', 'postXml' => $blankXml->asXML()]);
+$shopUrlId = (int) $createdShopUrl->shop_url->id;
+echo 'Successfully created shop url ' . $shopUrlId . PHP_EOL;
+```
+
+## Update your webservice key permissions
+
+At this point if you still use the same webservice key it will probably not have the permission to edit this new shop, you can edit your webservice key in the BackOffice and associate it with your new shop.
```

Comment:
```suggestion
At this point, if you still use the same webservice key, it will probably not have the permission to edit this new shop, you can edit your webservice key in the BackOffice and associate it with your new shop.
```

Reply:
No way to do it with API ?

Reply:
There's no API for Webservice key so no, not via API
It might be possible using the `WebserviceKey` object from the core, like it was made in the "Creating access" part, but it requires to have the core class accessible, along with the database set and all

In short it's probably doable in a module, but in a standalone script not that easy, I didn't take the time to setup an environment to do this, but it could an improvement to add later